Resources/Audio/DeltaV/Jukebox/attributions.yml

```diff
@@ -41,6 +41,11 @@
   copyright: "every light blinking at once by Sunbeamstress, converted to mono"
   source: "https://soundcloud.com/sunbeamstress/every-light-is-blinking-at-once"
 
+- files: ["Red_Vox_-_Garbage Land_MONO.ogg"]
+  license: "Custom"
```

Comment:
get a concrete license from them like CC-BY-NC-SA-3.0 (attribution, non commercial, share alike) which is similar to the loose terms given

Reply:
![image](https://github.com/user-attachments/assets/2ef884e6-c8ba-4dfe-96e1-b447427841a6)
[Garbage Land](https://vine.bandcamp.com/track/garbage-land)
![image](https://github.com/user-attachments/assets/07f76da3-2752-455c-8a54-0a030fbe9f38)
[Space Asshole](https://idlethumbs.bandcamp.com/track/space-asshole)
![image](https://github.com/user-attachments/assets/487b9e95-da95-4e54-9169-da51e3e30bdf)
[DOS=HIGH, UMB](https://masterbootrecord.bandcamp.com/track/dos-high-umb)
On the bandcamp pages for this song, Space Asshole, and DOS=HIGH, UMB, they're all already under an "all rights reserved" license, which is a concrete license, and all of them are put under Custom because they are all _used in the game with permission from the artist_.

Reply:
CC0-1.0 is the way you write it for that case